### ISSUE TITLE
BUG: a generic error occurred in GDI+ (waveform section distorted)

### ISSUE THREAD

**Issue by Mixkd** [SubtitleEdit/subtitleedit#5599]
Hello
I've been using Subtitle Edit 3.5.10 for a few months now. This is the first time I come across such an error. 12 minutes into subtitling the video, an error popped up saying "a generic error occurred in GDI+", as a result of which the waveform section got distorted (shown in the picture I attached). I tried closing it and opening it once again, but to no avail. I tried installing the latest version of the software, which is 3.6.4, but that didn't solve the issue either. I've not changed any settings regarding graphics whatsoever. 
Any help would be highly appreciated.
![2021-12-12 11_14_23-Window](https://user-images.githubusercontent.com/95998237/145704726-9adfc40f-ef9d-4d3f-afbf-8f0014e46d26.png)

**Comment by niksedk** [comment]
Try to use "mpv" as video player instead of VLC (see Options - Settings - Video player) and then restart your computer.

Does it still happen?
Does the problem only occur with one special subtitle or all subtitles?

**Comment by Mixkd** [comment]
I tried deleting the last two lines of the subtitle, and I didn't get the error anymore. I did so because the error displayed as soon as I clicked on either of the last two lines. 
It occurred only with this specific subtitle. I told the story to one of my friends, he said "I've got the same error several times for a couple of subtitles, but I never figured out what the issue was. I restarted my computer and the issue was solved."
